strawberryfields/gbs/data.py

```diff
@@ -61,8 +61,8 @@
 Molecules
 ---------
 
-Using the :mod:`~.gbs.vibronic` module, GBS data has been generated by encoding a variety of
-different molecules. The GBS samples can be used to recover the vibronic spectrum of the molecule.
+Using the :mod:`~.gbs.vibronic` module, GBS data has been generated for formic acid at zero
+temperature. The GBS samples can be used to recover the vibronic spectrum of the molecule.

```

Comment:
We decided to do just one molecule? Or add more in a follow up PR?

Reply:
Juan Miguel suggested having only one molecule.

Reply:
Just one molecule will fit all our needs and we're reaching the point were we need to wrap things up, so I indeed recommended having just formic acid. As usual, if we think it is important to add more, we can always do so in the future